src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -95,6 +97,10 @@ class MockClientStats {
   size_t num_async_stream_infer_calls{0};
   size_t num_start_stream_calls{0};
 
+  std::atomic<size_t> num_active_infer_calls{0};
```

Comment:
Should all of the other variables be updated to be atomics?

Reply:
Updated 👍